packages/terra-clinical-result-view/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-clinical-result-view",
+  "main": "lib/ResultView.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-clinical.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Clinical",
+    "terra-clinical-result-view",
+    "ResultView",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
```

Comment:
I think you can remove this peer dependency if there is a hard dependency specified below